adafruit_requests.py

```diff
@@ -324,9 +324,8 @@ def _throw_away(self, nbytes: int) -> None:
                 to_read -= self._recv_into(buf, to_read)
         remaining = nbytes % len_buf
         if remaining:
```

Comment:
Looking at it again, I think we can eliminate this `if remaining:` check also. What do you think?
I'm pretty new at working with github's UI so I'm not sure if I'm doing this right.

Reply:
You're right, it's redundant! I made a release but go ahead and make another PR, and I'll make another reelase.

Reply:
maybe `to_read` and `remaining` could have the same name.

Reply:
Sounds good. I'll work on it and see if it makes sense to combine the variables. It might be confusing though. I'll send another pull request in a bit, may not be today.